src/lib/pipeline/m2/index.js

```diff
@@ -377,11 +379,21 @@ class M2 extends THREE.Group {
   applyBillboards(camera) {
     for (let i = 0, len = this.billboards.length; i < len; ++i) {
       const bone = this.billboards[i];
-      this.applyBillboard(camera, bone);
+
+      switch (bone.userData.billboardType) {
+        case 0:
+          this.applySphericalBillboard(camera, bone);
+          break;
+        case 3:
+          this.applyCylindricalZBillboard(camera, bone);
+          break;
+        default:
+          break;
+      }
```

Comment:
Assuming we'll be adding support for X and Y once we find some models that actually have it? :smile: